index.js

```diff
@@ -37,9 +37,13 @@ if (argv.help || argv.h) {
   console.log('  -h, --help   show this help')
   console.log('  -q, --quiet  don\'t output stdout from tests unless an error occors')
   console.log('  --verbose    output a lot of information while running')
+  console.log('  -n           do a dry-run (don\'t actually execute anything)')
   console.log('  --compat     output just module version compatibility - no errors')
   console.log('  --ci         only run on CI servers when using .tav.yml file')
   process.exit()
+} else if (argv.version) {
```

Comment:
I'd prefer if we also allowed `-v`:

```suggestion
} else if (argv.version || argv.v) {
```

And also updated the help output above with a line like this:

```
  -v, --version   show version
```

Reply:
Will do. I initially avoided it because `-v` could be used as a short opt for `--verbose`.

Reply:
commit f4049f9